examples/quickstarts/helidon-quickstart-se/src/main/java/io/helidon/examples/quickstart/se/GreetService.java

```diff
@@ -100,6 +105,27 @@ private void sendResponse(ServerResponse response, String name) {
         response.send(returnObject);
     }
 
+    private static <T> T processErrors(Throwable ex, ServerRequest request, ServerResponse response) {
+                if (ex.getCause() instanceof JsonException){
+
+                LOGGER.log(Level.FINE, "Exception occurred while processing payload", ex);
+                    String error = String.format("Error with JSON: %s", ex.getCause().toString());
+                    JsonObject jsonErrorObject = JSON.createObjectBuilder()
+                        .add("error", error)
+                        .build();
+                    response.status(Http.Status.BAD_REQUEST_400).send(jsonErrorObject);
+                } else {
+
+                LOGGER.log(Level.FINE, "Exception occurred while processing payload", ex);
+                    JsonObject jsonErrorObject = JSON.createObjectBuilder() .add("error", ex.getLocalizedMessage()) .build();
+                    response.status(Http.Status.INTERNAL_SERVER_ERROR_500).send(jsonErrorObject);
+                    request.next(ex);
+                }
+
+                LOGGER.log(Level.FINE, "Exception occurred while processing payload", ex);
```

Comment:
You are effectively logging the exception twice. If the log message is not customized based on the exception type, then you can remove the first 2 and leave this one.